src/common/components/Inputs/InputExportKey.tsx

```diff
@@ -1,46 +1,41 @@
-import React from 'react'
+import React, { useState } from 'react'
 import SVG from 'react-inlinesvg'
+import { clipboard } from 'electron'
+
 import Input, { TInput } from './Input'
-import EyeIcon from '../../assets/icons/ico-eye.svg'
-import EyeIconHidden from '../../assets/icons/ico-eye-hidden.svg'
 import PasteIcon from '../../assets/icons/ico-paste.svg'
+import CheckIcon from '../../assets/icons/ico-check-green.svg'
```

Comment:
I think this check icon is already in the common/components. Please check!

Reply:
updated